### DISCUSSION TITLE
Reload SSL certs

### DISCUSSION THREAD

**Discussion by alxhotel** [uNetworking/uWebSockets#943] [Q&A] [answered]
What is the best way to reload the SSL certificates in production without downtime?

**Comment by ghost** [comment, marked as answer]
This has come up before. I don't know, easiest is to just restart servers. Otherwise I would gradually transition by starting new servers while closing old ones.

**Comment by alxhotel** [comment]
Thanks for the response!

I agree, gradually transitioning is the best way. I just thought that maybe uWebSockets could have something similar to what Nginx has, that reloads the configuration file.
